doc/threat_model/threat_model.md

```diff
@@ -188,6 +188,14 @@ TKey has been provisioned with:
 The configuration must have been written into the NVCM and
 locked by blowing the fuses.
 
+### unreleased
+
+#### Mitigations
+
+- Software attacks: 
+
+  - Access to RAM outside of physical RAM stops the CPU instead of
```

Comment:
Should we use a more description, and possibly more hard wording than 'stops'? Possibly 'force CPU into a trap state that requires a reset / reboot'?